Plugins/Covfie/src/FieldConversion.cpp

```diff
@@ -0,0 +1,208 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/Covfie/FieldConversion.hpp"
+
+namespace Acts::CovfiePlugin {
+
+namespace {
+
+/// @brief Get the value of the interpolated field at a specific position in
+/// min (inclusive) max (inclusive).
+/// @note The unchecked lookup allows the access to the field value at the max edges
+/// since the domain is typically min (inclusive) max (exclusive).
+/// @param magneticField the interpolated magnetic field.
+/// @param cache the magnetic field cache.
+/// @param position the position of the field to look up.
+/// @return the field value at the given position.
+Acts::Vector3 getFieldEdgeInclusive(
+    const Acts::InterpolatedMagneticField& magneticField,
+    [[maybe_unused]] Acts::InterpolatedMagneticField::Cache& cache,
+    const Acts::Vector3& position) {
+  // Check if position in within [min; max].
+  bool inBounds = position[0] <= magneticField.getMax()[0] &&
+                  position[1] <= magneticField.getMax()[1] &&
+                  position[2] <= magneticField.getMax()[2] &&
+                  position[0] >= magneticField.getMin()[0] &&
+                  position[1] >= magneticField.getMin()[1] &&
+                  position[2] >= magneticField.getMin()[2];
+  if (!inBounds) {
+    throw std::runtime_error{
+        "Field lookup failure (position not in [min, max])"};
+  }
+  return magneticField.getFieldUnchecked(position);
+}
+
+/// @brief Get the value of the field at a specific position of a general magnetic field.
+/// @param magneticField the interpolated magnetic field.
+/// @param cache the magnetic field cache.
+/// @param position the position of the field to look up.
+/// @return the field value at the given position.
+Acts::Vector3 getFieldEdgeInclusive(
```

Comment:
I don't fully understand what this function is for.

Reply:
Removed.